### PR TITLE
add chevrons to menu items

### LINKED ISSUE #8560: Chevrons to items in menu
image
- Let's add chevrons to items that open as a sidebar (fullwidth) and dont redirect to a different page (like learn, create) 

- this should apply to: notifications, shopping cart 

when going back from profile
image
- it should get you back to the menu items, like cart and notifications do

### PR DESCRIPTION
**Pull request by iMac7**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8560 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw&usdamount=undefined)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/koda_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot_20240319_165856

### CODE REVIEW

**Comment by yangwao** [comment]
cc @prury

**Comment by yangwao** [comment]
@prury can you check? 
@iMac7 can you please resolve code conflicts?

**Comment by prury** [comment]
> @prury can you check? @iMac7 can you please resolve code conflicts?

checking

**Comment by prury** [comment]
last thing: 
notification icon moved away from text: 

https://github.com/kodadot/nft-gallery/assets/36627808/d84bd4cd-af8c-41e9-ba4d-74fe07994fc6

**Comment by yangwao** [comment]
@kodadot/internal-dev please review

File: assets/styles/components/_connect-wallet.scss [hunk 1]
```diff
@@ -13,6 +13,10 @@
 }
 
 .connect-wallet-modal {
+  @include mobile {
+    z-index: 999 !important;
+  }
```
Comment (Jarsen136) [inline assets/styles/components/_connect-wallet.scss:18]: Could you try to use tailwind class like `sm:z-[999]`?

**Review by Jarsen136 — APPROVED** [review]
otherwise lgtm

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 9.57 USD/DOT ~ 3.135 $DOT
🧗 [14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw](https://polkadot.subscan.io/account/14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw)
🔗 [0xf9229ceb860e19dfca7d10afea4acd7290c344caab96872985221e33b56af813](https://polkadot.subscan.io/extrinsic/0xf9229ceb860e19dfca7d10afea4acd7290c344caab96872985221e33b56af813)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**